service-api/module/Application/src/Controller/HealthcheckController.php

```diff
@@ -61,4 +63,13 @@ public function healthCheckServiceAction(): JsonModel
             ]);
         }
     }
+
+    public function serviceAvailabilityAction(): JsonModel
+    {
+        $status = $this->ssmClient->getParameter([
+            'Name' => 'service-availability'
```

Comment:
This should use `AWS_SSM_SERVICE_AVAILABILITY`:
```suggestion
            'Name' => 'service-availability'
```